### PR TITLE
modifying paper and author networks

### PR DESCRIPTION
**Pull request by femalves**
**Summary of changes for author_network:** 

partition = community.best_partition(G, random_state=0)

return {“root”: top_level, “bibcode_dict”: bib_dict, “link_data”: sorted(link_data}) 

**Summary of changes for paper_network:** 

partition = community.best_partition(G, random_state=0)

count_references = sorted(references.items(), key=lambda x: (len(x[1], x[0]), reverse = True)[:5]

Proposed changes for the future include removing this part of the code: 

   # NOTE: We should remove this altogether and take the opportunity to change this in Nectar
    # NOTE: From Python 2 to 3 transition
    # Older networkx versions were producing a links structure using positional ids
    # so we created the artificial key 'stable_index' to make things easier on the front-end side
    # But newer networkx versions using node ids for the links structure, therefore 'stable_index'
    # has become redundant, but BBB uses it
    for i, n in enumerate(summary_json["nodes"]):
        #cache this so graph manipulation later is easier
        #summary_json["nodes"][i]["stable_index"] = i # For older networkx versions
        summary_json["nodes"][i]["stable_index"] = summary_json["nodes"][i]["id"]
     
Knowing this, I think it’s important to modify test_paper_network_resource to account for changes in the code. 

I also removed some lines of code that weren’t being used. 

**Full write-up**: https://docs.google.com/document/d/1wDe2nRVVgdh_D4w0raX5oUteRQqAbgG_022vws7rMRI/edit?pli=1

### CODE REVIEW

**Review by ehenneken — APPROVED** [review]

File: vis_services/tests/test_internals.py [hunk 1]
```diff
@@ -11,7 +11,16 @@
 from vis_services.lib import author_network
 from vis_services.lib import paper_network
 from vis_services.lib import tf_idf
+import logging
```
Comment (tjacovich) [inline vis_services/tests/test_internals.py:14]: It looks like you tore out all the logging statements in these tests after you were done. If so, it'd probably be worth removing the logger definitions here.

File: vis_services/tests/test_internals.py [hunk 2]
```diff
@@ -129,40 +133,9 @@ def get_group_references(group):
...
-        for x in test_js_paper_network['fullGraph']['links']:
-            if x['overlap'] == ['1988A&A...196...84C', '1985ApJ...299..211E']:
-                print(x)
-        mismatch_count = 0
-        for x in test_js_paper_network['fullGraph']['links']:
-           if x not in links_values:
-               mismatch_count += 1
-        print('fullGraph.links mismatch count: {}'.format(mismatch_count))
+
+        self.assertEqual(sorted(processed_data), sorted(test_js_paper_network))
```
Comment (tjacovich) [inline vis_services/tests/test_internals.py:138]: This works well for this, but in the future I think the underlying test class has a method called `assertUnsortedEqual` That might make this easier.

**Review by tjacovich — APPROVED** [review]
Looks good. One comment on the logging import in `test_internals` but once that's resolved this looks good to merge.

**Comment by tjacovich** [comment]
Your summary for the `vis` update looks good. Would you mind moving the full document to the RFC folder in the drive so everyone has access?

**Pull request merged**